ai-eval-service/evaluator.py

```
@@ -208,91 +147,44 @@ def _resolve_toxicity_classifiers(self) -> List[str]:
-        
         return {
-            "toxic_fraction": safe_float(toxicity_metrics.get('Toxic Fraction'), 0.0),
-            "expected_max_toxicity": safe_float(toxicity_metrics.get('Expected Maximum Toxicity'), 0.0),
-            "toxicity_probability": safe_float(toxicity_metrics.get('Toxicity Probability'), 0.0),
+            "toxic_fraction": _safe_float(toxicity_metrics.get('Toxic Fraction'), 0.0),
+            "expected_max_toxicity": _safe_float(toxicity_metrics.get('Expected Maximum Toxicity'), 0.0),
+            "toxicity_probability": _safe_float(toxicity_metrics.get('Toxicity Probability'), 0.0),
         }
     
-    def _format_stereotype_metrics(self, stereotype_metrics: Dict, category: str = "gender") -> Dict[str, float]:
+    def _format_stereotype_metrics(self, stereotype_metrics: dict[str, Any], category: str = "gender") -> dict[str, float]:
```

Ruff (0.14.10)

Unused method argument: `category`

(ARG002)